isisd/isis_sr.h

```diff
@@ -53,13 +53,26 @@
 
 #define SRGB_LOWER_BOUND               16000
 #define SRGB_UPPER_BOUND               23999
+#define SRLB_LOWER_BOUND               1000
```

Comment:
These are different from both the defaults in the provided yang model and the Cisco defaults which inspired the SRGB ones above, I'm guessing they should read 15000 and 15999 respectively

Reply:
Good catch. I forgot to update the #define

Reply:
Done